docs/performance/transformers-optimization.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Transformers Optimizer 
+parent: Performance
```

Comment:
Add a description here - this shows up in search and drives people to click (or not)

Reply:
Added